app/src/main/java/org/wikipedia/feed/FeedCoordinatorBase.java

```diff
@@ -301,7 +300,6 @@ private boolean isDailyCardType(@NonNull Card card) {
 
     private boolean shouldShowProgressCard(@NonNull FeedClient pendingClient) {
         return pendingClient instanceof SuggestedEditsFeedClient
-                || pendingClient instanceof AggregatedFeedContentClient
```

Comment:
@cooltey why was this removed?

Reply:
Because `AggregatedFeedContentClient` is not extended from `FeedClient`, the logic is unnecessary here at the beginning.